### PR TITLE
Error string should not be capitalized or end with punctuation

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

File: mold_test.go [hunk 1]
```diff
@@ -521,7 +521,7 @@ func TestTimeType(t *testing.T) {
 
 	err = set.Field(context.Background(), &tt, "default,dive")
 	NotEqual(t, err, nil)
-	Equal(t, err.Error(), "Invalid dive tag configuration")
+	Equal(t, err.Error(), "invalid dive tag configuration")
```
Comment (deankarn) [inline mold_test.go:524]: should probably change to errors.Is(err, ErrInvalidDive)` instead of string matching

Comment (frederikhors) [reply mold_test.go]: See my latest commit. Do you mean this?

Comment (deankarn) [reply mold_test.go]: no I mean using the built in `errors.Is` eg.
```go
Equal(t, errors.Is(err, ErrInvalidDive), true)
```

Comment (frederikhors) [reply mold_test.go]: Done. But what's the difference?

They both are working there.

1. `Equal(t, err, ErrInvalidDive)`
2. `Equal(t, errors.Is(err, ErrInvalidDive), true)`

Comment (deankarn) [reply mold_test.go]: The difference is one relies upon a type, the other a string message. Now if the message changes the test does not and it also supports wrapped errors.

**Pull request merged**